contracts/Insurance.sol

```diff
@@ -221,7 +216,7 @@ contract Insurance is IInsurance {
             return 0;
         }
 
-        uint256 ratio = PRBMathUD60x18.div(poolTarget - poolHoldings, levNotionalValue);
+        uint256 ratio = PRBMathUD60x18.pow(PRBMathUD60x18.div(poolTarget - poolHoldings, poolTarget), 2 * 10**18);
```

Comment:
Also, RBMath.pow(x, y) requires x > 1 in WAD (see https://github.com/hifi-finance/prb-math/blob/main/contracts/PRBMathUD60x18.sol#L357).

Since poolTarget - poolHoldings / poolTarget is always < 1, I'm not sure how this works. A test case as suggested in my other comment would check this.